test/integration/oauth_request_header_test.go

```diff
@@ -89,6 +95,11 @@ qLwYJxjzwYTLvLYPU5vHmdg8v5wIXh0TaRTDTdKViISGD09aiXSYzw==
 )
 
 func TestOAuthRequestHeader(t *testing.T) {
+	// Test data used by auth proxy
```

Comment:
I was expecting a new test.  Should this now be two tests?  Can you add a comment indicating all the things this is testing?